client/state/activity-log/actions.js

```diff
@@ -48,9 +48,10 @@ export function rewindActivateFailure( siteId ) {
 	};
 }
 
-export function activityLogRequest( siteId ) {
+export function activityLogRequest( siteId, params ) {
```

Comment:
Looks great. I've seen dot notation used for nested params [elsewhere](https://github.com/Automattic/wp-calypso/blob/5c925841da2b5840d3b4fa3b7cc432a3f7876c80/client/lib/query-manager/index.js#L215) which is quite nice.